### PR TITLE
Add list of fields in statistics doc

### PR DESCRIPTION
**Pull request by MartinKarp**
Added list of the different statistics that are computed in addition to some more doc. Relates to discussion #1130 and issue #1299

### CODE REVIEW

File: doc/pages/user-guide/statistics-guide.md [hunk 1]
```diff
@@ -3,6 +3,131 @@
...
+| `enabled`           | Whether to enable the statistics computation.                        | `true` or `false` | `true`        |
+| `start_time`        | Time at which to start gathering statistics.                         | Positive real     | 0             |
+| `sampling_interval` | Interval, in timesteps, for sampling the flow fields for statistics. | Positive integer  | 10            |
+
+In addition to the usual controls for the output, which then outputs the averages computes from the last time the statistics were written to file.
+
+For example, if one wants to sample the fields every 4 time steps and compute the averages in time intervals of 20 and write the output every 20 time units, and start collecting statistics after an initial transient of 50 time units the following would work:
+
+~~~~~~~~~~~~~~~{.json}
+statistics": {
```
Comment (timfelle) [inline doc/pages/user-guide/statistics-guide.md:45]:
```suggestion
"statistics": {
```

File: doc/pages/user-guide/statistics-guide.md [hunk 2]
```diff
@@ -3,6 +3,131 @@
 \tableofcontents
 
 Under development, updated incrementally
+
+Statistics in the context of Neko, is the common name for fields that are averaged in time and possible also space.
+
+The statistics module in Neko computes the temporal average of a wide range of fields.
+
+In the following page we use the following convetion for a field
+- \f$ u \f$, the instantaneous field.
```
Comment (timfelle) [inline doc/pages/user-guide/statistics-guide.md:12]:
```suggestion
In the following page we use the following convection for a field
```

File: doc/pages/user-guide/statistics-guide.md [hunk 3]
```diff
@@ -3,6 +3,131 @@
...
+In `mean_field` the following averages are stored. The stored in variable column is which field one finds the computed statistic if one opens the file in paraview or visit.
+
+| Number | Statistic | Stored in variable |
+| ------ | --------- | ------------------ |
+| 1 | \f$ \langle p \rangle \f$ | Pressure|
+| 2 | \f$ \langle u \rangle \f$ | X-Velocity|
+| 3 | \f$ \langle v \rangle \f$ | Y-Velocity|
+| 4 | \f$ \langle w \rangle \f$ | Z-Velocity|
+
+In `stats` several other statistics are stored, and while not all might be interesting to your specific usecase, with them most different budgets and quantities of interest can be computed. They are stored as the following:
```
Comment (timfelle) [inline doc/pages/user-guide/statistics-guide.md:65]:
```suggestion
In `stats` several other statistics are stored, and while not all might be interesting to your specific use case, with them most different budgets and quantities of interest can be computed. They are stored as the following:
```

File: doc/pages/user-guide/statistics-guide.md [hunk 4]
```diff
@@ -3,6 +3,131 @@
...
+e33 &= \left(\frac{\partial w}{\partial x}\right)^2 + \left(\frac{\partial w}{\partial y}\right)^2 + \left(\frac{\partial w}{\partial z}\right)^2 \\\\
+e12 &= \frac{\partial u}{\partial x}  \frac{\partial v}{\partial x} + \frac{\partial u}{\partial y}\frac{\partial v}{\partial y}+ \frac{\partial u}{\partial z}\\frac{\partial v}{\partial z} \\\\
+e13 &= \frac{\partial u}{\partial x}  \frac{\partial w}{\partial x} + \frac{\partial u}{\partial y}\frac{\partial w}{\partial y}+ \frac{\partial u}{\partial z}\\frac{\partial w}{\partial z} \\\\
+e23 &= \frac{\partial v}{\partial x}  \frac{\partial w}{\partial x} + \frac{\partial v}{\partial y}\frac{\partial w}{\partial y}+ \frac{\partial v}{\partial z}\\frac{\partial w}{\partial z} \\\\
+\end{aligned}
+$$
+
+
+# Postprocessing
+Of course, these statistics are only the "raw statistics" in the sense that in general we are not interested in \f$ \langle uu\rangle \f$, but rather say the rms of the velocity fluctuation. FOr this we need to posprocess the statistics.
```
Comment (timfelle) [inline doc/pages/user-guide/statistics-guide.md:125]:
```suggestion
Of course, these statistics are only the "raw statistics" in the sense that in general we are not interested in \f$ \langle uu\rangle \f$, but rather say the rms of the velocity fluctuation. FOr this we need to postprocess the statistics. 
```

File: doc/pages/user-guide/statistics-guide.md (hunk 2 above)
Comment (MartinKarp) [reply doc/pages/user-guide/statistics-guide.md:12]: Im not sure I understand this one.

Comment (timfelle) [reply doc/pages/user-guide/statistics-guide.md:12]: I have no idea why it did that 😅. I tried to flag the typo: `convetion`

**Review by njansson — APPROVED** [review]

**Comment by njansson** [comment]
@timfelle can we merge?

**Review by timfelle — APPROVED** [review]
> @timfelle can we merge? 

Sure, i guess its a good start. Lets just get the information out to everyone.

**Pull request merged**